src/diagonal.works/b6/ingest/worlds.go

```diff
@@ -0,0 +1,43 @@
+package ingest
+
+import (
+	"diagonal.works/b6"
+)
+
+var DefaultWorldFeatureID = b6.FeatureID{
+	Type:      b6.FeatureTypeCollection,
+	Namespace: "diagonal.works/world",
+	Value:     0,
+}
+
+type Worlds interface {
+	FindWorld(id b6.FeatureID) MutableWorld
+}
+
+type MutableWorlds struct {
+	Base    b6.World
+	Mutable map[b6.FeatureID]MutableWorld
+}
+
+func (m *MutableWorlds) FindWorld(id b6.FeatureID) MutableWorld {
+	if !id.IsValid() {
+		id = DefaultWorldFeatureID
+	}
+	w, ok := m.Mutable[id]
+	if !ok {
+		if m.Mutable == nil {
+			m.Mutable = make(map[b6.FeatureID]MutableWorld)
```

Comment:
y :( y create in find

Reply:
Renamed FindOrCreateWorld. I don't want to separate for the moment, as I'd have to add functions to check if a world exists, etc.